src/crimson/os/seastore/lba_manager/btree/btree_lba_manager.cc

```diff
@@ -311,7 +432,7 @@ _init_cached_extent(
       LOG_PREFIX(BtreeLBAManager::init_cached_extent);
       if (!iter.is_end() &&
 	  iter.get_key() == logn->get_laddr() &&
-	  iter.get_val().paddr == logn->get_paddr()) {
+	  iter.get_val().pladdr.get_paddr() == logn->get_paddr()) {
```

Comment:
I still think that this should be:
```c++
if (!iter.is_end() &&
    iter.get_key() == logn->get_laddr() &&
    iter.get_val().pladdr.is_paddr() &&
    iter.get_val().pladdr.get_paddr() == logn->get_paddr()) {
```
in case that `e` is an outdated extent but `e->get_laddr()` happens to be a valid key for an irrelavent indirect map.

Reply:
yeah.